api/server/models/optionModel.js

```diff
@@ -3,6 +3,7 @@ var mongoose = require("mongoose");
 var optionSchema = mongoose.Schema({
   time: Date,
```

Comment:
Mentioned above, split this into start (Date) and end (Date). Or to keep consistency with what Leo did in another model do startTime (Date) and endTime (Date).